lib/helpers/acceptance.js

```diff
@@ -7,7 +7,6 @@ var existsSync        = require('exists-sync');
 var Promise           = require('rsvp');
 var runCommand        = require('./run-command');
 var tmp               = require('./tmp');
-var conf              = require('./conf');
```

Comment:
@sivakumar-kailasam these seem like unrelated changes. could you rebase this branch on top of the current `master`? sorry for the review delay, forgot about the PR... 😞